src/main/resources/i18n/Messages_en.properties

```diff
@@ -306,6 +306,11 @@ search.facilityInventories.germinatingQuantity=Germinating quantity (facility in
 search.facilityInventories.notReadyQuantity=Not Ready quantity (facility inventory)
 search.facilityInventories.readyQuantity=Ready quantity (facility inventory)
 search.facilityInventories.totalQuantity=Total quantity (facility inventory)
+search.facilityInventoryTotals.germinatingQuantity=Germinating quantity (total facility inventory)
```

Comment:
I can't recall how/where the search.table.field strings get used.

Reply:
> I can't recall how/where the search.table.field strings get used.

They are the column headers in CSV exports.